src/Features/CSharp/Portable/SimplifyTypeNames/SimplifyTypeNamesCodeFixProvider.cs

```diff
@@ -40,9 +40,6 @@ protected override string GetTitle(string diagnosticId, string nodeText)
                 case IDEDiagnosticIds.SimplifyMemberAccessDiagnosticId:
                     return string.Format(CSharpFeaturesResources.Simplify_member_access_0, nodeText);
 
-                case IDEDiagnosticIds.RemoveQualificationDiagnosticId:
-                    return CSharpFeaturesResources.Remove_this_qualification;
-
```

Comment:
this was not hittable.  SimplifyTypeNamesCodeFixProvider derives from AbstractSimplifyTypeNamesCodeFixProvider which only registers the following as fixable IDs:

```c#
        public sealed override ImmutableArray<string> FixableDiagnosticIds { get; } =
            ImmutableArray.Create(
                IDEDiagnosticIds.SimplifyNamesDiagnosticId,
                IDEDiagnosticIds.SimplifyMemberAccessDiagnosticId,
                IDEDiagnosticIds.PreferBuiltInOrFrameworkTypeDiagnosticId);
```